TASK-8 - [FE] Tela de buscar voos

PaG0Dezaum2005 opened the pull request:
https://trello.com/c/0CowinYX/8-fe-tela-consultar-extrato-de-milhas

PaG0Dezaum2005 committed:
feat: adicona as rotas, router nos botoes e no header

PaG0Dezaum2005 committed:
fix: adiciona userId e store de userId. Havia a necessidade de filtrar as reservas de cada usuário

PaG0Dezaum2005 committed:
feature: Adiciona mock de extrato, adiciona store de usuario, adiciona compnente de visualizacao de extrato

PaG0Dezaum2005 committed:
fix: remove debug

Ital0Alve5 commented on front/src/stores/user.ts:
```diff
@@ -0,0 +1,12 @@
+import { defineStore } from 'pinia'
+
+export const useUserInfoStore = defineStore('userInfo', {
```
passar pra setup

Ital0Alve5 commented:
fix: mesmo tendo o mock de extratos, a tabela não mostra resultados
![image](https://github.com/user-attachments/assets/413b515a-3049-4e92-b882-8d0f1e4a8300)

Ital0Alve5 commented on front/src/views/Client/Miles/MilesExtract.vue:
```diff
@@ -0,0 +1,194 @@
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              <template v-if="table.getRowModel().rows?.length">
```
Esse if sempre está dando falso e está caindo no "no results". O mock está sendo consumido?

PaG0Dezaum2005 committed:
fix: troca stores de options pra setup

PaG0Dezaum2005 committed:
fix: reseta form depois de dar submit, envia para o mock, ajeita mock

PaG0Dezaum2005 committed:
fix: ajeita mock de funcionarios para ter os atributos userId e miles

YasmimZedelinski reviewed (changes requested):
- Lista sumindo ao dar refresh na página
- Botão de ir para a próxima página não está funcionando depois de comprar milhas
- A quantidade de milhas atual (após a compra) não está refletindo na página inicial da tabela e nem na página de compras

PaG0Dezaum2005 committed:
fix: -ajeitei ordenacao por valor
-ajeitei tabela ao atualizar pagina
-ajeites valores de 'minhas milhas' e 'saldo de milhas'
-ajeitei botao 'proxixmo' na tabela nao funcionar (problema esta no toast do shadcn que bloqueia os botoes de funcionar)
